package.json

```diff
@@ -32,5 +32,8 @@
     "@babel/core": "^7.1.5",
     "@babel/preset-env": "^7.1.5",
     "babel-plugin-add-module-exports": "^1.0.0"
+  },
+  "dependencies": {
+    "d2l-hypermedia-constants": "^6.71.0"
```

Comment:
Is this adding this repo as a dependency for itself or am I completely misreading this bit here?

Reply:
@marcsoiferman  good catch...removed it